crates/compiler/builtins/bitcode/src/main.zig

```diff
@@ -264,6 +264,9 @@ comptime {
     if (builtin.target.cpu.arch == .aarch64) {
         @export(__roc_force_setjmp, .{ .name = "__roc_force_setjmp", .linkage = .Weak });
         @export(__roc_force_longjmp, .{ .name = "__roc_force_longjmp", .linkage = .Weak });
+    } else if (builtin.os.tag == .windows) {
+        @export(__roc_force_setjmp_windows, .{ .name = "__roc_force_setjmp", .linkage = .Weak });
+        @export(__roc_force_longjmp_windows, .{ .name = "__roc_force_longjmp", .linkage = .Weak });
```

Comment:
Does this mean we won't have support for Arm64 and Windows combined? 

Maybe the zig implementation for setjmp/longjmp be ok for Arm64 Windows? 

I don't have an arm64 Windows device to test against, and I'm not sure how popular that combination is.

Reply:
it's very rare, I'd expect more stuff to break in that case anyway so we'll cross that bridge when the hardware is more common